examples/data-objects/webflow/src/test/.mocharc.js

```diff
@@ -0,0 +1,43 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
```

Comment:
no need to do it in this PR, but we probably want this shareable

Reply:
Yep... that's the plan.